doc/changes/changes_0.2.0.md

```diff
@@ -1,6 +1,10 @@
-# Server for the Exasol Test-Setup Abstraction 0.2.0, released 2022-??-??
+# Server for the Exasol Test-Setup Abstraction 0.2.0, released 2022-05-02
 
-Code name: Fixed failing startup when docker is not running on localhost
+Code name: Add BucketFS support
+
+## Summary
+
+This release adds support for accessing BucketFS, allows creating a DB connection with autocommit deactivated and fixes the failing startup when docker is not running on localhost.
```

Comment:
Go support is not mentioned. Ticket is not mentioned.

Reply:
✅